repo/packages/S/spark/0/package.json

```diff
@@ -12,5 +12,12 @@
     "icon-large": "https://downloads.mesosphere.io/spark/assets/icon-service-spark-large.png"
   },
   "preInstallNotes": "We recommend a minimum of two nodes with at least 2 CPU and 2GB of RAM available for the Spark Service and running a Spark job. Also the Spark client might take up to a few minutes to download",
-  "postInstallNotes": "Spark DCOS Service has been successfully installed!\nNote that the service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nWe recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for use by the Spark Service.\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK"
+  "postInstallNotes": "Spark DCOS Service has been successfully installed!\nNote that the service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.\nWe recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for use by the Spark Service.\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK",
+  "postUninstallNotes": "The Apache Spark DCOS Service has been uninstalled and will no longer run.\nPlease follow the instructions at http://beta-docs.mesosphere.com/services/spark/#uninstall to clean up any persisted state.",
```

Comment:
postInstallNotes should have Spark DCOS Service has been successfully installed!\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK